example/src/GalleryPage.react.js

```diff
@@ -12,10 +12,12 @@ import json from "./data/Gallery.Items";
 
 function GalleryPage(): React.Node {
   const options = (
-    <Form.Select className="mr-2">
-      <option value="asc">Newest</option>
-      <option value="desc">Oldest</option>
-    </Form.Select>
+    <React.Fragment>
```

Comment:
@burakakyol the input didn't quite make it into the Fragment, maybe a missing commit?

Reply:
My bad, forgot to save file :)